templates/sub/project_settings/{{ cookiecutter.__folder_name }}/frontend/packages/{{ cookiecutter.frontend_addon_name }}/src/config/settings.ts

```diff
@@ -0,0 +1,10 @@
+import type { ConfigType } from "@plone/registry";
+
+export default function install(config: ConfigType) {
+  // Language settings
+  config.settings.isMultilingual = false;
+  config.settings.supportedLanguages = ["{{ cookiecutter.language_code }}"];
+  config.settings.defaultLanguage = "{{ cookiecutter.language_code }}";
```

Comment:
@ericof @sneridagh We don't want this in addons! If I install an addon in my project I don't expect it to change the language.

In Volto 19 this isn't necessary, because Volto gets the languages from the backend.

Reply:
Sorry, I missed that this is in project_settings. It's okay then, but it would be better not to have it in Volto 19.